examples/detailedlist/detailedlist/app.py

```diff
@@ -9,11 +11,13 @@ class ExampleDetailedListApp(toga.App):
     def on_select_handler(self, widget, row, **kwargs):
         self.label.text = 'You selected row: {}'.format(row) if row is not None else 'No row selected'
 
-    def on_refresh_handler(self, widget, **kwargs):
+    async def on_refresh_handler(self, widget, **kwargs):
+        self.label.text = 'Refreshing list...'
+        await asyncio.sleep(1)
```

Comment:
Lol wut. Fake refresh time?

Reply:
Yes - it's a demo app; without a background server to get actual data, we need *something* to fake the load time. I should add a comment to explain though.